ajax/libs/cytoscape-panzoom/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "cytoscape-panzoom",
+  "version": "2.1.3",
+  "description": "Panzoom extension for Cytoscape.js",
+  "filename": "cytoscape-panzoom.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cytoscape/cytoscape.js-panzoom.git"
+  },
+  "keywords": [
+    "cytoscape",
+    "cyext"
+  ],
+  "license": "LGPL-3.0+",
```

Comment:
Unreliable `+` should be deprecated.
https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json#L119